README.md

```diff
@@ -12,7 +12,7 @@ This code is free and publicly available under the Apache 2.0 open source licens
 
 ## Why Hummingbot?
 
-* **Both CEX and DEX connectors**: Hummingbot supports connectors to centralized exchanges like Binance and KuCoin, as well as decentralized exchanges like Uniswap and PancakeSwap on various blockchains (Ethereum, BNB Chain, etc).
+* **Both CEX and DEX connectors**: Hummingbot supports connectors to centralized exchanges like Binance and KuCoin, as well as decentralized exchanges like Amm and PancakeSwap on various blockchains (Ethereum, BNB Chain, etc).
```

Comment:
That's not needed, uniswap is connector (exchange) not strategy here

Reply:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed